SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -742,7 +787,9 @@ bool SpatialGDKGenerateSchema()
 
 	// Generate Schema for classes loaded in memory.
 
-	if (!SpatialGDKGenerateSchemaForClasses(GetAllSupportedClasses()))
+	TArray<UObject*> AllClasses;
+	GetObjectsOfClass(UClass::StaticClass(), AllClasses);
```

Comment:
This definitely returns the same set of classes as the class iterator? (just making sure)

Reply:
Yes, I've found such call to `GetObjectsOfClass` in Iterator constructor, so it's exactly the same code.